src/objects/index.ts

```diff
@@ -1,25 +1,27 @@
-import { createApplication, application } from '@/objects/application'
-import { createBox } from '@/objects/box'
-import { createScales } from '@/objects/scales'
-import { setStage } from '@/objects/stage'
-import { createViewport } from '@/objects/viewport'
+import { startApplication, stopApplication } from '@/objects/application'
+import { startBox } from '@/objects/box'
+import { emitter } from '@/objects/events'
+import { startScales } from '@/objects/scales'
+import { startStage, stopStage } from '@/objects/stage'
+import { startViewport } from '@/objects/viewport'
 
 export * from './application'
 export * from './stage'
 export * from './viewport'
 export * from './scales'
 
 export function start(stage: HTMLDivElement): void {
-  setStage(stage)
+  startApplication()
+  startViewport()
+  startScales()
+  startBox()
 
-  createApplication()
-  createViewport()
-  createScales()
-  createBox()
+  startStage(stage)
 }
 
 export function stop(): void {
-  application.destroy(true, {
-    children: true,
-  })
+  emitter.all.clear()
+
+  stopApplication()
+  stopStage()
```

Comment:
I wonder if emitting some sort of global stop event might be nice. Like I believe the sprites will need to be destroyed to avoid memory leaks? Things could be different in Pixi 7 though.

Reply:
We totally could. Though I believe when I'm destroying the app I'm also telling it to destroy the children.